pkg/kubectl/cmd/get/get.go

```diff
@@ -327,7 +412,7 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 			if o.PrintWithOpenAPICols {
 				return
 			}
-			if o.ServerPrint && o.IsHumanReadablePrinter && !o.Sort {
+			if o.ServerPrint && o.IsHumanReadablePrinter {
```

Comment:
When printing we should be requesting the full object, so that you can perform that json field matching, am I missing something?

Reply:
don't we need to add something like:
```
if o.Sort {
  req.Param("includeObject", "Object")
}
```

Reply:
and above, if we're sorting, need to disable chunking:

```diff
--- a/pkg/kubectl/cmd/get/get.go
+++ b/pkg/kubectl/cmd/get/get.go
@@ -310,6 +310,10 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
        if o.PrintWithOpenAPICols && o.ServerPrint {
                fmt.Fprintf(o.IOStreams.ErrOut, "warning: --%s requested, --%s will be ignored\n", useOpenAPIPrintColumnFlagLabel, useSe
        }
+       chunkSize := o.ChunkSize
+       if o.Sort {
+               chunkSize = 0
+       }

        r := f.NewBuilder().
                Unstructured().
@@ -318,7 +322,7 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
                LabelSelectorParam(o.LabelSelector).
                FieldSelectorParam(o.FieldSelector).
                ExportParam(o.Export).
-               RequestChunksOf(o.ChunkSize).
+               RequestChunksOf(chunkSize).
                IncludeUninitialized(o.IncludeUninitialized).
                ResourceTypeOrNameArgs(true, args...).
                ContinueOnError().
```

Reply:
> and above, if we're sorting, need to disable chunking:

Either that or we could warn users to do that. Both options seems viable.

Reply:
We should disable chunking when you request sorting, and include object.